rskj-core/src/test/java/co/rsk/peg/BridgeSupportTest.java

```diff
@@ -536,6 +537,49 @@ void getProposedFederationAddress_whenProposedFederationExists_shouldReturnAddre
             assertEquals(expectedAddress, actualProposedFederationAddress.get());
         }
 
+        @ParameterizedTest
+        @EnumSource(FederationMember.KeyType.class)
+        void getProposedFederatorPublicKeyOfType_whenBridgeSupportReturnsEmpty_shouldReturnEmpty(FederationMember.KeyType keyType) {
+            // Arrange
+            var index = 1;
+
+            // Act
+            var actualProposedFederatorPublicKey = bridgeSupport.getProposedFederatorPublicKeyOfType(index, keyType);
+
+            // Assert
+            assertFalse(actualProposedFederatorPublicKey.isPresent());
+        }
+
+        @ParameterizedTest
+        @EnumSource(FederationMember.KeyType.class)
+        void getProposedFederatorPublicKeyOfType_whenProposedFederationExists_shouldReturnAddress(FederationMember.KeyType keyType) {
+            FederationMember member = federation.getMembers().get(0);
```

Comment:
Avoid literals if they repeat in more than one place. Set into a variable.

Reply:
Updated